### ISSUE TITLE
Normalisation for pretrained models?

### ISSUE THREAD

**Issue by gohweixun** [xjtu-omics/msisensor-rna#5]
Hello,

May I know what was the normalisation used for the training data, for the 4 pretrained models that have been included in the repository? Thank you.
